lightningd/peer_control.c

```diff
@@ -1169,22 +1169,15 @@ static void json_dev_forget_channel(struct command *cmd, const char *buffer,
 	struct dev_forget_channel_cmd *forget = tal(cmd, struct dev_forget_channel_cmd);
 	forget->cmd = cmd;
 
-	/* If &forget->force is used directly in p_opt_def() below then
-	 * gcc 7.3.0 fails with:
-	 * 'operation on ‘forget->force’ may be undefined [-Werror=sequence-point]'
-	 *
-	 * See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=86584
-	 *
-	 * Hence this indirection.
-	 */
```

Comment:
The indirection is now necessary because the callback now requires an "address of pointer".  So I removed the comment.

Reply:
Yep, great.